api/api_sources/sources/database/database-schema/schema-sqls/SpecificUseCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: specific_use_code ### --
+
+        
+CREATE TABLE specific_use_code ();
+ALTER TABLE specific_use_code ADD COLUMN specific_use_code_id SERIAL PRIMARY KEY;
+ALTER TABLE specific_use_code ADD COLUMN specific_use_code VARCHAR(2) NOT NULL UNIQUE;
+ALTER TABLE specific_use_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE specific_use_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE specific_use_code IS 'Code table for schema of Specific use codes of the observation';
```

Comment:
I'd like to recommend we add a more descriptive table comment that describes "specific use code" from a business perspective.

For example; The specific use code provides details regarding the use of an area where an observation has been made. This assists in determining the appropriate response and treatment methods when dealing with the observation. Values are GR (Gravel pit), NS (No-spray buffer), NH (Numbered Highway), PL (Parking Lot), RT (Rec Site/Trail), RA (Rest Area), SS (Sensitive Site), TL (Transfer station/Landfil), WP (Within PFZ), YW (Yard/Ditching waste dump).

Reply:
Thanks! 
I will update SQL file